WebAuthnCredentialMixin using incorrect validation syntax

Introduced in https://github.com/pypi/warehouse/pull/5795/files#diff-ad2285487bd44546b51ec41ca45cb0b5fb1b6ce16156b70a88fbdd0313920c55R55

Currently:
https://github.com/pypi/warehouse/blob/1f48c86a85371dca020bee549f5f912783e67f69/warehouse/accounts/forms.py#L107-L108

Syntax _should_ be `credential = wtforms.StringField(validators=[wtforms.validators.InputRequired()])`

Updating this line causes 3 tests to fail:

```
FAILED tests/unit/accounts/test_forms.py::TestWebAuthnAuthenticationForm::test_credential_valid - AssertionError: {'credential': ['This field is required.']}
FAILED tests/unit/accounts/test_forms.py::TestWebAuthnAuthenticationForm::test_credential_bad_payload - AssertionError: assert equals failed
FAILED tests/unit/accounts/test_forms.py::TestWebAuthnAuthenticationForm::test_credential_invalid - AssertionError: assert equals failed
```

<details>

```python
=========================================================================== FAILURES ===========================================================================
_____________________________________________________ TestWebAuthnAuthenticationForm.test_credential_valid _____________________________________________________

self = <tests.unit.accounts.test_forms.TestWebAuthnAuthenticationForm object at 0xffff75ef01d0>

    def test_credential_valid(self):
        request = pretend.stub()
        challenge = (pretend.stub(),)
        origin = (pretend.stub(),)
        rp_id = (pretend.stub(),)
        form = forms.WebAuthnAuthenticationForm(
            request=request,
            credential=json.dumps({}),
            user_id=pretend.stub(),
            user_service=pretend.stub(
                verify_webauthn_assertion=pretend.call_recorder(
                    lambda *a, **kw: ("foo", 123456)
                )
            ),
            challenge=challenge,
            origin=origin,
            rp_id=rp_id,
        )

        assert form.challenge is challenge
        assert form.origin is origin
        assert form.rp_id is rp_id
>       assert form.validate(), str(form.errors)
E       AssertionError: {'credential': ['This field is required.']}
E       assert False
E        +  where False = <bound method Form.validate of <warehouse.accounts.forms.WebAuthnAuthenticationForm object at 0xffff7477d4d0>>()
E        +    where <bound method Form.validate of <warehouse.accounts.forms.WebAuthnAuthenticationForm object at 0xffff7477d4d0>> = <warehouse.accounts.forms.WebAuthnAuthenticationForm object at 0xffff7477d4d0>.validate

tests/unit/accounts/test_forms.py:969: AssertionError
__________________________________________________ TestWebAuthnAuthenticationForm.test_credential_bad_payload __________________________________________________

self = <tests.unit.accounts.test_forms.TestWebAuthnAuthenticationForm object at 0xffff75ef08d0>
pyramid_config = <pyramid.config.Configurator object at 0xffff708204d0>

    def test_credential_bad_payload(self, pyramid_config):
        request = pretend.stub()
        form = forms.WebAuthnAuthenticationForm(
            request=request,
            credential="not valid json",
            user_id=pretend.stub(),
            user_service=pretend.stub(),
            challenge=pretend.stub(),
            origin=pretend.stub(),
            rp_id=pretend.stub(),
        )
        assert not form.validate()
>       assert (
            str(form.credential.errors.pop())
            == "Invalid WebAuthn assertion: Bad payload"
        )
E       AssertionError: assert equals failed
E         'This field is required.'                  'Invalid WebAuthn assertion: Bad payload'

tests/unit/accounts/test_forms.py:984: AssertionError
____________________________________________________ TestWebAuthnAuthenticationForm.test_credential_invalid ____________________________________________________

self = <tests.unit.accounts.test_forms.TestWebAuthnAuthenticationForm object at 0xffff75ef0f90>

    def test_credential_invalid(self):
        request = pretend.stub(remote_addr="127.0.0.1")
        user = pretend.stub(
            record_event=pretend.call_recorder(lambda *a, **kw: None),
        )
        user_service = pretend.stub(
            get_user=pretend.call_recorder(lambda userid: user),
            verify_webauthn_assertion=pretend.raiser(
                AuthenticationRejectedError("foo")
            ),
        )
        form = forms.WebAuthnAuthenticationForm(
            request=request,
            credential=json.dumps({}),
            user_id=1,
            user_service=user_service,
            challenge=pretend.stub(),
            origin=pretend.stub(),
            rp_id=pretend.stub(),
        )
        assert not form.validate()
>       assert form.credential.errors.pop() == "foo"
E       AssertionError: assert equals failed
E         'This field is required.'  'foo'

tests/unit/accounts/test_forms.py:1010: AssertionError
```

</details>

I didn't know exactly the right path forward, so decided to at least log this.

--- a/tests/unit/accounts/test_forms.py
+++ b/tests/unit/accounts/test_forms.py
@@ -950,8 +950,8 @@ class TestWebAuthnAuthenticationForm:
         origin = (pretend.stub(),)
         rp_id = (pretend.stub(),)
         form = forms.WebAuthnAuthenticationForm(
+            formdata=MultiDict({"credential": json.dumps({})}),
             request=request,
-            credential=json.dumps({}),
             user_id=pretend.stub(),
             user_service=pretend.stub(
                 verify_webauthn_assertion=pretend.call_recorder(
@@ -972,8 +972,8 @@ class TestWebAuthnAuthenticationForm:
     def test_credential_bad_payload(self, pyramid_config):
         request = pretend.stub()
         form = forms.WebAuthnAuthenticationForm(
+            formdata=MultiDict({"credential": "not valid json"}),
             request=request,
-            credential="not valid json",
             user_id=pretend.stub(),
             user_service=pretend.stub(),
             challenge=pretend.stub(),
@@ -998,8 +998,8 @@ class TestWebAuthnAuthenticationForm:
             ),
         )
         form = forms.WebAuthnAuthenticationForm(
+            formdata=MultiDict({"credential": json.dumps({})}),
             request=request,
-            credential=json.dumps({}),
             user_id=1,
             user_service=user_service,
             challenge=pretend.stub(),

--- a/warehouse/accounts/forms.py
+++ b/warehouse/accounts/forms.py
@@ -105,7 +105,7 @@ class TOTPValueMixin:
 
 
 class WebAuthnCredentialMixin:
-    credential = wtforms.StringField(wtforms.validators.InputRequired())
+    credential = wtforms.StringField(validators=[wtforms.validators.InputRequired()])
 
 
 class RecoveryCodeValueMixin:

